src/models/calendarObject.js

```diff
@@ -151,9 +151,29 @@ export default class CalendarObject {
 	getAllObjectsInTimeRange(start, end) {
 		const iterator = this.vcalendar.getVObjectIterator()
 		const firstVObject = iterator.next().value
-		if (!firstVObject) {
+		if (!firstVObject || (firstVObject.name === 'VTODO' && !firstVObject.dueTime)) {
 			return []
 		}
+		if (firstVObject.name === 'VTODO') {
+			// calendar-js is missing isAllDay, canModifyAllDay for TodoComponents
+			firstVObject.isAllDay = function() {
```

Comment:
This is an ok workaround for now, but ideally we would fix this in [calendar-js](https://github.com/georgehrke/calendar-js). Let me take care of that: https://github.com/georgehrke/calendar-js/issues/73